pkg/server/backuprestoreserver.go

```diff
@@ -210,6 +216,7 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 			}
 			miscellaneous.SleepWithContext(ctx, retryTimeout)
 		}
+		peerUrlTLSEnabled = true
```

Comment:
I think it's not safe to assume that `peerUrlTLS` is enabled when using multi-node is used. It's true for Gardener but not in general.

Reply:
Ok you are correct.

Reply:
changed it